content/en/observability_pipelines/setup/_index.md

```diff
@@ -393,6 +393,29 @@ For a deeper dive into recommendations for deploying and scaling multiple Worker
 - See [Deployment Design and Principles][7] for information on what to consider when designing your Observability Pipelines architecture.
 - See [Best Practices for OPW Aggregator Architecture][8] for details on the recommended Observability Pipelines aggregator architecture, which is optimized for scaling.
 
+## Deployment Modes
+
+<div class="alert alert-warning">
+  Remote configuration for Observability Pipelines is in private beta. Contact <a href="https://docs.datadoghq.com/help/">Datadog support</a> or your Customer Success Manager for access.
+</div>
+If you are enrolled in the private beta of [Remote Configuration][7] to remotely roll out changes to your OP Workers, you remotely roll out changes to your Workers from the Datadog UI, rather than making updates to your pipeline configuration in a text editor and rolling out your changes manually. You can choose your deployment method when you create a pipeline and install your Workers.
+
+After deploying a pipeline, you can also switch deployment methods, such as going from manually managed pipeline to remote configuration enabled pipeline or vice versa. 
+
+If you want to switch from a remote configuration deployment to a manually managed deployment:
+1. Navigate to Observability Pipelines.
+2. Click the settings cog.
+3. In **Deployment Mode**, select **manual** to enable it.
+4. Set the flag, `DD_OP_REMOTE_CONFIGURATION_ENABLED` to `false` and restart the Worker. Workers that aren't restarted with this flag will continue to be remote configuration enabled, which means that the Worker will not be updated manually through a local configuration file
+
+If you want to switch from manually managed deployment to a remote configuration deployment:
+1. Navigate to Observability Pipelines.
+2. Click the settings cog.
+3. In **Deployment Mode**, select **Remote Configuration** to enable it.
+4. Set the flag, `DD_OP_REMOTE_CONFIGURATION_ENABLED` to `true` and restart the Worker. Workers that aren't restarted with this flag will not poll for configurations deployed in the UI. 
+6. Deploy a version in your version history, so that the Workers receive the new version configuration. Click on a version, click _Edit as Draft_, then click _Deploy_. 
```

Comment:
```suggestion
6. Deploy a version in your version history, so that the Workers receive the new version configuration. Click on a version. Click **Edit as Draft** and then click **Deploy**. 
```